lib/subjects/remover.rb

```diff
@@ -56,11 +59,19 @@ def has_been_collected_or_classified?
       !orphan_subject
     end
 
+    def belongs_to_other_subject_set?
+      return false unless subject_set_id != nil
+
+      orphan_subject.set_member_subjects.where.not(subject_set_id: subject_set_id).count.positive?
+    end
+
     def has_been_talked_about?
       panoptes_client.discussions(
         focus_id: subject_id,
         focus_type: 'Subject'
       ).any?
+    rescue TypeError => _e
```

Comment:
I don't think we need this TypeError. If the reason you are adding this is because of a failing spec, I think we should look back at how we are stubbing the response.